src/Sulu/Bundle/ContentBundle/Markup/LinkTag.php

```diff
@@ -64,11 +66,9 @@ public function __construct(
     /**
      * {@inheritdoc}
      */
-    public function parseAll($attributesByTag)
+    public function parseAll(array $attributesByTag, $locale)
     {
         $request = $this->requestStack->getCurrentRequest();
-        $locale = $request->getLocale();
```

Comment:
Don't exactly know why you changed that, but if we remove that line we also don't need the preceding line, do we?